pkg/operator/latencyprofilecontroller/controller.go

```diff
@@ -115,12 +119,59 @@ func (c *LatencyProfileController) sync(ctx context.Context, syncCtx factory.Syn
 	uniqueRevisions := []int32{}
 	for _, nodeStatus := range operatorStatus.NodeStatuses {
 		revision := nodeStatus.CurrentRevision
+
+		// during day-0: pre-empt
+		if revision == 0 {
+			_, err = c.updateStatus(
+				ctx,
+				false, false,
+				reasonClusterInit,
+				"one or more static pod revision(s) are unknown",
+			)
+			return err
+		}
+
```

Comment:
As per issues described in [this comment](https://github.com/openshift/library-go/pull/1372/#issuecomment-1145732452):
- if we find a currentRevision that has value 0, we take a different step back
  - ensures: that we do not encounter `WorkerLatencyProfileDegraded` due to an error saying `config-0 config map not found`.

Reply:
Follow-up: https://github.com/openshift/library-go/pull/1372/#discussion_r897146939, https://github.com/openshift/library-go/pull/1372/#discussion_r897148395.